consensus/src/experimental/buffer.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::experimental::hashable::Hashable;
```

Comment:
there is a `Hash` trait from std

Reply:
I feel this is slightly different from std::hash::Hash, we precisely want HashValue as the key.